common/build.gradle.kts

```diff
@@ -38,6 +38,14 @@ publishing {
     }
 }
 
+sourceSets{
+    test {
```

Comment:
It'd be better to move the common assets that we use in multiple places, like HelloWorld and DesignSwitcher, to a common directory, but I'd rather not risk a change like that while I can't check to make sure I didn't break anything.